docs/admin/kubeadm.md

```diff
@@ -84,6 +84,9 @@ Valid values are the ones supported by `controller-manager`, namely `"aws"`,
 the cloud provider, you should create a `/etc/kubernetes/cloud-config`
 file manually, before running `kubeadm init`. `kubeadm` automatically
 picks those settings up and ensures other nodes are configured correctly.
+Format and content of the `/etc/kubernetes/cloud-config` file depends on
```

Comment:
"The exact format and content of the file `/etc/kubernetes/cloud-config` depends on the type you specified for `--cloud-provider`; see the appropriate documentation for your cloud provider for details."